packages/editor/src/lib/exports/html-to-image/README.md

```diff
@@ -0,0 +1,16 @@
+# html-to-image
+
+The code in this folder has been very roughly adapted from html-to-image:
```

Comment:
so since you reviewed i ran into some further issues with how we were doing things before - reading & writing in the same pass was causing the browser to constantly recalculate layout etc. 

I've since reworked the code here futher, and at this point there's pretty much zero left from html-to-image, so I've removed this entirely. I can bring it back if you think it's worth it though